src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngKey.Export.cs

```diff
@@ -32,10 +32,11 @@ public byte[] Export(CngKeyBlobFormat format)
                 throw errorCode.ToCryptographicException();
 
             byte[] buffer = new byte[numBytesRequired];
-            errorCode = Interop.NCrypt.NCryptExportKey(_keyHandle, IntPtr.Zero, format.Format, IntPtr.Zero, buffer, numBytesRequired, out numBytesRequired, 0);
+            errorCode = Interop.NCrypt.NCryptExportKey(_keyHandle, IntPtr.Zero, format.Format, IntPtr.Zero, buffer, buffer.Length, out numBytesRequired, 0);
```

Comment:
Which file are you referring to? The only one’s I could find were the ones where I was passing a pinned pointer down (and thus has no Length property, and passing down the source array’s Length didn’t seem any more robust than passing down “numBytesRequired.”)